lib/v2/juejin/events.js

```diff
@@ -0,0 +1,44 @@
+const got = require('@/utils/got');
+
+module.exports = async (ctx) => {
+    const city = ctx.params.city === 'all' ? '' : ctx.params.city;
+    let cityName = '';
+
+    const response = await got({
+        method: 'post',
+        url: 'https://api.juejin.cn/event_api/v1/event/event_list',
+        json: {
+            city: city === 'all' ? '' : city,
+            count: 20,
+            cursor: 0,
+        },
+    });
+
+    const items = response.data.data.map((item) => {
+        const content = item.title;
+        const title = content;
+        const guid = item.id;
```

Comment:
This guid is way too easy for collision
You can use something like
```suggestion
        const guid = `juejin:events:${city ? `${city}:` : ''}${item.id}`;
```
instead